### PR TITLE
Use helm client insead of running helm pod

### PR DESCRIPTION
**Pull request by saheienko**
Add helm client, that helps to get rid of the "helm job pod". 
A structure for helm repositories was updated. Also, apps (releases) with all statuses are now displayed.

Used packages:
```
k8s.io/kubernetes/helm@v2.8.2
```

### CODE REVIEW

File: pkg/core/helm_releases.go [hunk 1]
```diff
@@ -139,204 +154,59 @@ func (c *HelmReleases) Delete(id *int64, m *model.HelmRelease) ActionInterface {
...
+				if err != nil {
+					return errors.Wrap(err, "build helm client")
 				}
+
+				resp, err := hclient.DeleteRelease(m.Name, helm.DeletePurge(true))
+				if err != nil {
+					return errors.Wrapf(err, "delete %s release", m.Name)
+				}
+
+				c.Core.Log.Debugf("DEBUG - Delete %s release: %s", m.Name, resp.Info)
```
Comment (yegorch) [inline pkg/core/helm_releases.go]: @saheienko Could you please change logging statements to logrus?

File: pkg/helm/client.go [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+)
+
+var (
+	tillerPodLabels = labels.Set{"app": "helm", "name": "tiller"}
+)
+
+// TODO: remove after refactoring
+// keep track of the created clients
+var (
+	clients map[string]*client
```
Comment (yegorch) [inline pkg/helm/client.go]: @saheienko  Have you thought about use case  when that map is accessed/modified concurrently?

**Comment by yegorch** [comment]
@saheienko Any tests?

File: pkg/core/helm_releases.go (hunk 1 above)
Comment (saheienko) [reply pkg/core/helm_releases.go]: Updated

File: pkg/helm/client.go (hunk 2 above)
Comment (saheienko) [reply pkg/helm/client.go]: Updated, use sync.Map here.

**Comment by saheienko** [comment]
@yegorch, integration tests will be added later.

**Pull request merged**